Issue page revamp for NRAI (#19790)
newrelic/docs-website: src/content/docs/alerts/incident-management/Issues-and-Incident-management-and-response.mdx

      src="/images/accounts_screenshot-crop_new-issue-page.webp"
    />

</Collapser>

## Causal analysis

<Callout title="preview">
  We're still working on this feature, but we'd love for you to try it out!

  If you have questions or feedback, or if you need help during the preview of the .NET agent's <DNT>**Instrumentation**</DNT> editor, send an email to [dotnet-li-editor-beta@newrelic.com](mailto:dotnet-li-editor-beta@newrelic.com).

  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
</Callout>

The causal analysis engine identifies potential symptoms that might have triggered an alert event and suggests immediate mitigating actions to address them.

> Consider a scenario where a PHP application encounters a memory leak, leading to a failure in the throughput SLI and triggering an alert. Our engine investigates by moving from the service level to the APM application and then to the infrastructure container to detect the symptom. 

Reviewer: ```suggestion
Consider a scenario where a PHP application encounters a memory leak that leads to a failure in the throughput SLI and triggers an alert. The engine investigates by moving from the service level to the APM application and then to the infrastructure container to detect the symptom. 
```

### How does the engine work?

> The causal analysis engine uses distinct analysis categories, such as deployment events, infrastructure resource limits, and more. Each category is designed to address various potential sources of anomalies and performance issues. These categories focus on specific data types and metrics, enabling precise analysis and more accurate identification of causal relationships.

Reviewer: ```suggestion
The causal analysis engine uses distinct analysis categories such as deployment events, infrastructure resource limits, and more. Each category is designed to address various potential sources of anomalies and performance issues. Each category focuses on specific data types and metrics, enabling precise analysis and more accurate identification of causal relationships.
```

> At the moment, we’re primarily focused on APM entity causal analysis. In the near term, our plan is to include infrastructure, Browser, among other entity types. 

Reviewer: Do we want to commit to this in docs?

### Mitigating actions & visualizations

> For every identified potential cause, the engine offers tailored mitigation actions that guide users through the necessary steps to quickly restore services and entities to their normal operational states. We recognize that many of our customers typically rely on NRQL to analyze significant queries, hence we provide relevant visuals alongside the underlying query for each cause.

Reviewer: ```suggestion
For every identified potential cause, the engine offers tailored mitigation actions that guide users through the necessary steps to quickly restore services and entities to their normal operational states. Since many of our customers rely on NRQL to analyze significant queries, we provide relevant visuals alongside the underlying query for each cause.
```

### Confidence scores

> In certain situations, you may encounter multiple potential causes. To assist you in prioritizing which cause to examine first, we provide confidence scores. These scores are categorized as low, medium, or high confidence.

Reviewer: ```suggestion
In certain situations, you may encounter multiple potential causes. To assist you in prioritizing which cause to examine first, New Relic provides confidence scores. These scores are categorized as low, medium, or high confidence.
```

### NR AI generated analysis

> In some scenarios, our causal engine may not identify an algorithm-driven cause. However, we have insights that can be utilized with LLMs to offer you actionable steps. Customers interested in this capability must have the New Relic AI entitlement enabled.

Reviewer: ```suggestion
In some scenarios, the causal engine may not identify an algorithm-driven cause. However, the insights can be used with LLMs to offer you actionable steps. To use this capability, enable the New Relic AI entitlement.
```

To further reduce noise or get improved incident correlation, you can change or customize your decisions. Decisions determine how incidents are grouped together.

> To get started, see [Decisions](/docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-decisions).

Reviewer: ```suggestion
To get started, refer [Decisions](/docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-decisions).
```
